### PR TITLE
Add serialVersionUID from 1.16 for AWSCredentialsImpl class

### PR DESCRIPTION
**Pull request by mattstep**
@andresrc @stephenc

### CODE REVIEW

File: src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java [hunk 1]
```diff
@@ -56,6 +56,8 @@
 
 public class AWSCredentialsImpl extends BaseAmazonWebServicesCredentials implements AmazonWebServicesCredentials {
 
+    private static final long serialVersionUID = -8073785642456657271L;
```
Comment (jtnord) [inline src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java]: Is this the correct serialVersionUID?  I was expecting `-3167989896315282034L`

Comment (mattstep) [inline src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java]: I generated it using the generator in intellij from the 1.16 release tag. How'd you generate yours?

Comment (jtnord) [inline src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java]: So it is what came from the serialization error log and what I got when I ran serialver

```
java.io.InvalidClassException: com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl; local class incompatible: stream classdesc serialVersionUID = -3167989896315282034, local class serialVersionUID = 5122525060396971762
```

```
C:\Users\jnord>c:\Java\jdk1.8.0_91\bin\serialver.exe -classpath C:\Users\jnord\Desktop\aws-credentials.jar;C:\Users\jnord\Desktop\classes;c:\Users\jnord\Desktop\jenkins-core-1.596.1.jar;c:\Users\jnord\Desktop\aws-java-sdk-core-1.10.16.jar com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl
com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl:    private static final long serialVersionUID = -3167989896315282034L;
```

**Comment by stephenc** [comment]
:bee: now that you are using the `serialver` tools version

**Pull request merged**